kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/k8s/listeners/KubeSimpleListener.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.systemtests.k8s.listeners;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.fabric8.kubernetes.client.dsl.ExecListener;
+
+/**
+ * The Kube listener.
+ */
+public class KubeSimpleListener implements ExecListener {
```

Comment:
`LoggingKubeListener` might be a beter name.
Do we really need to keep this?

Reply:
done. Thanks!